src/plots/VolumeVTK9/VolumeAttributes.code

```diff
@@ -446,9 +446,14 @@ Definition:
 // Creation:   June 18, 2003
 //
 // ****************************************************************************
+
+#ifdef VIEWER
+#include <avtCallback.h>
```

Comment:
Ok, so yeah these mid-file `#includes` are coming from the `.code` file. I don't think there is much that really needs to be done about it then. i think its fine as is.